Python/README.md

```diff
@@ -129,6 +129,10 @@ _add list here_
 - [Merging Sorted arrays](other/merge_sorted_arrays.py)
 - [Reverse a string using stack](other/Reverse_string_using_stack.py)
 - [Russian Peasant Algorithm](other/Russian_Peasant_Algorithm.py)
+- [Satellite Image Analysis using numpy](other/Satellite_Image_Analysis_using_numpy.ipynb)
+- [Swap Two Nibbles](other/Swap_two_nibbles.py)
+- [Unique Number III](other/Unique_number_III.py)
```

Comment:
Only add link of your file here. Remove unnecessary changes.

Reply:
Done Sir @atarax665